default.nix

```diff
@@ -95,6 +95,8 @@ let commonBuildInputs = pkgs:
     pkgs.obelisk
     pkgs.perl
     pkgs.removeReferencesTo
+    pkgs.which
```

Comment:
these are for figuring out why aarch64-linux doesn't want to build static executables.